### ISSUE TITLE
Accessible name is not defined for the "Runtime" combo box in add trigger or action panel 

### ISSUE THREAD

**Issue by Xiaoyu-Huang** [Azure/LogicAppsUX#5166]
### Describe the issue 

Repro:

1. go to any workflow in Azure portal 
2. click add trigger or action to open the side panel. 
3. The Runtime box and Action type box are missing Accessible name 

Issue: 

 Ensures buttons have discernible text (button-name - [https://accessibilityinsights.io/info-examples/web/button-name](<https://accessibilityinsights.io/info-examples/web/button-name>))

![image](<https://github.com/user-attachments/assets/5970a6d9-4778-4dfc-99f7-13363177915f>)

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Edge

### Additional context 

*No response*
